packages/text-editor/src/components/extension/nodeHighlight.ts

```diff
@@ -12,6 +12,11 @@ interface NodeHighlightExtensionOptions extends NodeUuidOptions {
   isHighlightModeOn: () => boolean
 }
 
+// eslint-disable-next-line @typescript-eslint/no-invalid-void-type
+function isRange (range: Range | undefined | null | void): range is Range {
```

Comment:
why not boolean response?

Reply:
I need this type guard because `getMarkRange` return type is `Range | void` and then I get the typescript error later that `Range | void` type does not have properties `to` and `from`.

This type guard just to tell typescript it is `Range` type (not void), as I can not add `&& range !== void`

Reply:
But return type is boolean?

Reply:
May be you have unintended upgrade of typescript in QMS?

Reply:
Some very weird concept actually, I do not think we need it here.

Reply:
@haiodo Removed, can you please review?